### PR TITLE
Update emscripten Dockerfile.

### PR DESCRIPTION
**Pull request by ekpyron**
Let's see if this works.

### CODE REVIEW

**Comment by cameel** [comment]
Looks good to me and checks passed. Are we merging it?

**Comment by ekpyron** [comment]
@cameel The process is usually:
- Open a draft with this to build a docker image - that worked here, but the test runs here still use the old docker image - so the docker build itself performed one build of soljson.js, but no tests and the other CI test runs do not yet use it.
- Open a second PR that updates the hash of the docker image to the one posted by the bot. I'm about to do that :-). The CI runs in *that* PR then accurately judge the change.
- If CI passes in the second PR, merge both.

So bear with me ;-).

**Comment by cameel** [comment]
Ah, ok, I know the process. I just thought there was something more holding it back. Like an unsolved problem or something.

**Comment by ekpyron** [comment]
> Ah, ok, I know the process. I just thought there was something more holding it back. Like an unsolved problem or something.

No, I was just in a call when the build passed :-).

**Review by cameel — APPROVED** [review]
In that case I'm going to approve it already since there doesn't seem to be anything wrong with it.

File: scripts/docker/buildpack-deps/Dockerfile.emscripten [hunk 1]
```diff
@@ -39,8 +43,8 @@ RUN set -ex; \
 	mkdir build; \
 	cd build; \
 	emcmake cmake \
+		-DCMAKE_INSTALL_PREFIX=$(em-config CACHE)/sysroot/usr \
```
Comment (axic) [inline scripts/docker/buildpack-deps/Dockerfile.emscripten:46]: Can you pull this out into a config variable so that the two copies are kept in sync?

Comment (ekpyron) [reply scripts/docker/buildpack-deps/Dockerfile.emscripten:46]: I probably could... but do you think that warrants rebuilding everything :-)?
It's a bug that it's required here anyways, so I'm pretty sure we'll need to change this on the next update anyways.

Comment (ekpyron) [reply scripts/docker/buildpack-deps/Dockerfile.emscripten:46]: I actually thought about adding a comment about removing it here next time (or at least trying to), but wasn't sure how nicely that would play with the line break escape, so I added the one on top.

Comment (ekpyron) [reply scripts/docker/buildpack-deps/Dockerfile.emscripten:46]: Also there's little danger updating this incorrectly - if it's not done right, the builds will fail.

Comment (axic) [reply scripts/docker/buildpack-deps/Dockerfile.emscripten:46]: I'm not too set on it.

**Review by axic — APPROVED** [review]

**Pull request merged**